src/bento/components/bento-reddit/OWNERS

```diff
@@ -0,0 +1,10 @@
+// For an explanation of the OWNERS rules and syntax, see:
+// https://github.com/ampproject/amp-github-apps/blob/main/owners/OWNERS.example
+
+{
+  rules: [
+    {
+      owners: [{name: 'ampproject/wg-components'}, {name: 'pablos1'}],
```

Comment:
who is pablos1? and can we just remove this and create an owners file for src/bento/components that lists wg-components as the owner?

Reply:
That would be a copy pasta error from the bento-jwplayer owners file. Right now each amp-component has its own OWNERS, Im not sure if there is a structural reason to keep it that way though.
@alanorozco What do you think?